impl/src/main/java/org/ehcache/impl/internal/DefaultTimeSourceService.java

```diff
@@ -33,7 +33,7 @@ public DefaultTimeSourceService(TimeSourceConfiguration config) {
     if (config != null) {
       timeSource = config.getTimeSource();
     } else {
-      timeSource = SystemTimeSource.INSTANCE;
+      timeSource = TickingTimeSource.INSTANCE;
```

Comment:
I'm not convinced this should be the default, and I'm not sure how we should be configuring it.

Reply:
It is configurable. But if it works, I think it should be the default since it's the faster. The only sad thing I see about it is the background thread. Then, I did a hardcoded one. I don't know how slower will be a configurable version. But yes, configuring the `GRANULARITY_IN_MS` and `PERIOD_BETWEEN_SYSTEM_SET_IN_MS` would make sense.

Reply:
And having a constant means the thing is spinning and not closeable for everyone